src/common/interfaces/IGatewayHandlers.sol

```diff
@@ -283,13 +283,13 @@ interface IBalanceSheetGatewayHandler {
         AssetId assetId,
         address receiver,
         uint128 amount,
-        D18 pricePerUnit,
+        D18 priceAssetPerShare,
         Meta calldata m
     ) external;
 
-    function triggerIssueShares(PoolId poolId, ShareClassId scId, address to, D18 pricePerShare, uint128 shares)
+    function triggerIssueShares(PoolId poolId, ShareClassId scId, address to, D18 priceAssetPerShare, uint128 shares)
         external;
 
-    function triggerRevokeShares(PoolId poolId, ShareClassId scId, address from, D18 pricePerShare, uint128 shares)
+    function triggerRevokeShares(PoolId poolId, ShareClassId scId, address from, D18 pricePoolPerShare, uint128 shares)
```

Comment:
Curious, what's the reason prices are different here? (Sorry if I missed a current thread about it). Is it due the asymetry we have in deposit/redemption flows?

Reply:
Great catch! This was indeed an incorrect naming, both expect the `pricePoolPerShare` because they lead to an increment/decrement of shares on the Hub side based using the provided price as `navPerShare`

Reply:
[398512f](https://github.com/centrifuge/protocol-v3/pull/184/commits/398512f66af49a2331a9f8c6496d8ec9b0764322)